app/swagger/swagger/requests/form1010_ezrs.rb

```diff
@@ -33,11 +35,6 @@ class Form1010Ezrs
               key :$ref, :Form1010EzrSubmissionResponse
             end
           end
-
-          response 422 do
-            key :description, 'Unprocessable Entity: one or more form validations has failed'
-            schema '$ref': :Errors
-          end
         end
```

Comment:
Removed this and added the `Swagger::Responses::ValidationError` instead (DRY'd it up a bit).